src/main/java/com/exasol/adapter/jdbc/ResultSetMetadataReader.java

```diff
@@ -102,6 +102,16 @@ private String createColumnDescriptionFromDataTypes(final List<DataType> types)
         return types;
     }
 
+    private void validateMetadata(final ResultSetMetaData metadata) {
+        if (metadata == null) {
+            throw new RemoteMetadataReaderException(ExaError.messageBuilder("E-VS-COM-JDBC-34") //
```

Comment:
```suggestion
            throw new RemoteMetadataReaderException(ExaError.messageBuilder("F-VS-COM-JDBC-34") //
```

If it's a ticket mitigation the user probably can't do anything about it? Then I would use `F`